paddle/cinn/backends/codegen_c_test.cc

```diff
@@ -65,16 +66,18 @@ TEST(CodeGenC, module) {
   target.os = Target::OS ::Linux;
   Module::Builder builder("module1", target);
 
-  auto stages = CreateStages({A, B, C});
-  auto func = Lower("add1", stages, {A, B, C});
+  ast_gen_ius::TensorGroup tensor_group({A, B, C});
+  auto func = lang::LowerToAst("add1", {A, B, C}, &tensor_group);
+
+  LOG(INFO) << "Huihuang debug: " << func << std::endl;
```

Comment:
This is the debug output I forgot to remove, I will delete it along with reviewer's comment.